Dockerfile-alpine

```diff
@@ -0,0 +1,48 @@
+FROM alpine:3.4
```

Comment:
note, this is called `Dockerfile-alpine` to be configured in Docker Hub to build separately